### DISCUSSION TITLE
PHP 8 life changing

### DISCUSSION THREAD

**Discussion by cavasinf** [kevinpapst/TablerBundle-Demo#13] [Q&A]
Do we allow PHP 8 syntax now it's forced to >=8.0.

I mean, when I see this:
https://github.com/kevinpapst/TablerBundle-Demo/blob/d5f98c5abc2ee400b099a3ca0667f1aded033f8f/src/EventSubscriber/MenuBuilderSubscriber.php#L21-L27

I really want to change it to this:
```php
 //private $security; 
  
 public function __construct(
    private AuthorizationCheckerInterface $security
 ) { }
```

**Comment by kevinpapst** [comment]
Sure, at least in the demo that is totally fine.
